cypress-custom/support/commands.js

```diff
@@ -53,7 +53,7 @@ function enterOutputAmount(tokenAddress, amount, selectToken = false) {
 }
 
 function stubResponse({ url, alias = 'stubbedResponse', body }) {
-  cy.intercept({ method: 'GET', url }, _responseHandlerFactory(body)).as(alias)
+  cy.intercept({ method: 'POST', url }, _responseHandlerFactory(body)).as(alias)
```

Comment:
why this change?

Reply:
Quote endpoint is a POST

Reply:
well, yes, but the command is `stubResponse`.. should u add some parameter or something?

Reply:
:+1: done